src/custom/tsconfig-paths-custom.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "paths": {
       "@src/*": ["*"],
-      "*": ["custom/*", "*"]
+      "*": ["custom/*", "*"],
+      "@uniswap/*": ["custom/@uniswap/*"]
```

Comment:
Why this change? I don't see any note regarding it. Also I don't see that directory...

Reply:
good eye, should be removed i was testing stuff